config.lua

```diff
@@ -77,3 +77,16 @@ Config.Peds = {
         }
     }
 }
+
+Config.Employment = {
+    disable = false, -- Set to true to disable the employment menu
```

Comment:
```suggestion
    enabled = true, -- Set to false to disable the employment menu
```
The positive is usually clearer to understand than the negative. That way you can write "if enabled" rather than "if not disabled"